### PR TITLE
[Not for merge] News87: PR Review Club summaries

### PR DESCRIPTION
**Pull request by jnewbery**
Throwing this up for early review and feedback. This PR is not for merge, and this section will be incorporated into Dave's News87 PR when it gets opened.

@harding suggested including these summaries in IRC:

```
16:52 < harding> jonatack also suggested mentioning it more and I've been really impressed with some of the pre-meeting material prepared in the past couple monhs.
16:53 < harding> Maybe we could have a regular montly section like the StackExachange section that summarizes the topics discussed during the previous month (with links).
16:53 < jnewbery> harding: I think that's a good idea, and I'm happy to contribute that write-up if others think it's worthwhile
```

I think this also addresses @jonatack's suggestion of highlighting review in #301.

Looking for feedback on length/style of this first summary. If y'all like it I'll write similar summaries for the other PR Review Club meetings this month.

### CODE REVIEW

File: _posts/en/newsletters/2020-03-04-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+contributors to the Bitcoin Core project to learn about how to take part in the
+review process. An experienced Bitcoin Core contributor provides background
+information about a selected Pull Request and then leads a discussion about the
+PR on IRC.
+
+The Review Club is an excellent way to learn about the Bitcoin protocol, the
+Bitcoin Core reference implementation, and the process of making changes to
+Bitcoin. Notes, questions and meeting logs are posted on the website for each
+meeting for those who are unable to attend in real time, and as a permanent
+resource for those wishing to learn about the Bitcoin Core development process.
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/en/newsletters/2020-03-04-newsletter.md]: Absolutely agree!

[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/en/newsletters/2020-03-04-newsletter.md]: Suggested wording if this becomes a monthly feature:

_The [Bitcoin Core PR Review Club](https://bitcoin-core-review-club.github.io/) is a weekly IRC meeting for newer Bitcoin Core contributors to learn about the project and the review process. In this monthly section, we summarize some recent Review Club meetings._

File: _posts/en/newsletters/2020-03-04-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    and benchmarking; and how to test and review changes to consensus-critical
+    code.
+
+[Bitcoin Core PR Review Club]:https://bitcoin-core-review-club.github.io/
+
+## Notable code and documentation changes
+
+{% include references.md %}
+{% include linkers/issues.md issues="16902,17977,18002" %}
+[review club 16902]: https://bitcoincore.reviews/16902.html
```
Comment (jonatack) [inline _posts/en/newsletters/2020-03-04-newsletter.md]: could leave off the .html for a nicer url

**Comment by jnewbery** [comment]
>  I think that's a bit long; I suggest trying to keep the average meeting bullet under 100 words

Did some cutting and got it down to 115 words. Do you think that's too long?

> Also, I suggest that we add "PR" to the unintroduced abbreviations section of the style guide

Agree. Done in a separate commit.

File: _posts/en/newsletters/2020-03-04-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+In this section, we summarise some recent Review Club meetings.
+
+- **[O(1) OP_IF/NOTIF/ELSE/ENDIF script implementation / Abstract out script execution out of VerifyWitnessProgram()][review club 16902]:**
+  These PRs ([Bitcoin Core #16902][] and [Bitcoin Core #18002][]) are
+  both small refactors that have been pulled out of the [Schnorr/Taproot
+  implementation PR][Bitcoin Core #17977]. The author explains in PR 18002:
+  "[this] simplifies the changes for Taproot significantly [...] I may as well
+  give it some additional attention by PRing it independently."
+
+    Both PRs make changes to the Bitcoin Core's script interpreter code -- a
```
Comment (jonatack) [inline _posts/en/newsletters/2020-03-04-newsletter.md]: "to the Bitcoin Core" or "to Bitcoin Core's" (derp, I didn't see it on the first pass)

**Comment by harding** [comment]
> got it down to 115 words. Do you think that's too long?

That sound good.  I think aiming for 100 words per bullet is a good guideline, but it shouldn't be a hard rule.

File: _posts/en/newsletters/2020-03-04-newsletter.md (hunk 3 above)
Comment (jnewbery) [reply _posts/en/newsletters/2020-03-04-newsletter.md]: fixed

File: _posts/en/newsletters/2020-03-04-newsletter.md (hunk 2 above)
Comment (jnewbery) [reply _posts/en/newsletters/2020-03-04-newsletter.md]: done

File: _posts/en/newsletters/2020-03-04-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    section of the codebase which is very seldom changed by developers. Discussion
+    in meeting covered Bitcoin Core's use of asserts and logging; performance
+    and benchmarking; and how to test and review changes to consensus-critical
+    code.
+
+[Bitcoin Core PR Review Club]:https://bitcoin-core-review-club.github.io/
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/en/newsletters/2020-03-04-newsletter.md]: done

File: _posts/en/newsletters/2020-03-04-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  These PRs ([Bitcoin Core #16902][] and [Bitcoin Core #18002][]) are
+  both small refactors that have been pulled out of the [Schnorr/Taproot
+  implementation PR][Bitcoin Core #17977]. The author explains in PR 18002:
+  "[this] simplifies the changes for Taproot significantly [...] I may as well
+  give it some additional attention by PRing it independently."
+
+    Both PRs make changes to the Bitcoin Core's script interpreter code -- a
+    section of the codebase which is very seldom changed by developers. Discussion
+    in meeting covered Bitcoin Core's use of asserts and logging; performance
+    and benchmarking; and how to test and review changes to consensus-critical
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/en/newsletters/2020-03-04-newsletter.md]: done

**Comment by jnewbery** [comment]
Fixed all review comments. I'm going to close this PR now and push a commit to Dave's PR when he opens it.

**Pull request closed without merging**